GOVERNANCE.md

```diff
@@ -1,131 +1,187 @@
+- Helping [design interfaces](https://openrefine.org/docs/technical-reference/introduction) for OpenRefine
 
 How to become an OpenRefine contributor? You will find more details in our [contributing guideline](https://github.com/OpenRefine/OpenRefine/blob/master/CONTRIBUTING.md)
 
 ### Committers
-If you make regular contribution to OpenRefine, you will most likely become a Committer.
+If you make regular contributions to OpenRefine, you will most likely become a Committer.
 
-Committers have earned enough trust from the community to review and merge pull requests.
+Committers have earned enough trust from the community to
 
-Therefore Committers:
-- Help contributors via the [developer discussion list](https://forum.openrefine.org/c/dev/8).
 - Triage issues, pull requests and [projects](https://github.com/OpenRefine/OpenRefine/projects)
 - Have direct access to the code base to create new branches
-- Organize the wiki
-- Review and merge pull requests submitted by contributors
+- Organize the GitHub wiki
+- Review pull requests submitted by contributors
+- Run the CI pipeline on your PRs automatically
 - Are part of the OpenRefine organization and have the OpenRefine badge on their GitHub profile
 
-#### How to become a Committer?  
+**How to become a Committer?**: Committers are voting-binding members; they select and elect new members through a voting process. Nominations should be submitted to the [developer discussion list](https://forum.openrefine.org/c/dev/8), where contributors can nominate themselves or other contributors. See the Member Election section below for more details.
+
+**Current list of Committers**: The list is available here: https://github.com/orgs/OpenRefine/people.
 
-Be a contributor and be nominated as a Committer. Current Committers select and elect new Committers. You may nominate yourself. Nomination should be sent to the [developer discussion list](https://forum.openrefine.org/c/dev/8)
+### Core Developer Group
+Also referred to as Core Dev. The Core Developer Group are trusted developers who
+- Merge pull requests
+- Publish releases
+- Review security vulnerability reports
+- Provide long-term project vision with the community
+- Are responsible for the technical direction of the project
+- Are part of the Admin team for the project on GitHub
 
-#### Current list of Committers
-The list is available here: https://github.com/orgs/OpenRefine/people.
+**How to join the Core Dev Group?**. To be eligible to serve on the Core Developers, individuals must first be Committers of the project. Core Dev Group members are voting-binding members; they select and elect new members through a voting process. Nominations should be submitted to the [developer discussion list](https://forum.openrefine.org/c/dev/8), where contributors can nominate themselves or other contributors. See the Member Election section below for more details.
```

Comment:
```suggestion
**How to join the Core Dev Group?**. To be eligible to serve on the Core Developers, individuals must first be Committers of the project. Core Dev Group members are voting-binding members; they select and elect new members through a voting process. Nominations should be submitted to the [developer discussion list](https://forum.openrefine.org/c/dev/8), where committers can nominate themselves or other committers. See the Member Election section below for more details.
```
Small change since the hierarchy is that only committers can become part of core dev, and not contributors directly who must first be committers then part of ccontributors. Change enforces that there cannot be any direct promotions into core dev by contributers.

Reply:
Good catch! That makes it more clear indeed.

Reply:
Thanks @thadguidry